### PR TITLE
Tranz vr

### PR DESCRIPTION
**Pull request by theRealKLH**
Adding TranzVR to Wankz scraper. put "cover" image back into wankzvr and removed blank images from tranzvr

### CODE REVIEW

File: pkg/scrape/wankz.go [hunk 1]
```diff
@@ -61,10 +61,13 @@ func WankzVRSite(wg *sync.WaitGroup, updateSite bool, knownScenes []string, out
...
 		// Cover URLs
 		for _, x := range []string{"cover", "hero"} {
-			if scraperID == "milfvr" && x == "cover" {
-				continue // MilfVR does not have a "cover" image unlike WankzVR
+			if (scraperID == "milfvr" && x == "cover") || (scraperID == "tranzvr" && x == "hero") {
+				continue // MilfVR does not have a "cover" image & TranzVR doesn't have a "hero" image
 			}
 			tmpCover := "https://cdns-i." + scraperID + ".com/" + sc.SiteID[0:1] + "/" + sc.SiteID[0:4] + "/" + sc.SiteID + "/" + x + "/large.jpg"
+			if scraperID == "tranzvr" {
+			tmpCover = "https://images.tranzvr.com/" + sc.SiteID[0:1] + "/" + sc.SiteID[0:4] + "/" + sc.SiteID + "/550/" + x + ".webp"
```
Comment (SpedNSFW) [inline pkg/scrape/wankz.go:69]: Line should be indented.

File: pkg/scrape/wankz.go [hunk 2]
```diff
@@ -74,8 +77,11 @@ func WankzVRSite(wg *sync.WaitGroup, updateSite bool, knownScenes []string, out
 			size = "1280"
 		}
 		for _, x := range []string{"1", "2", "3", "4", "5", "6"} {
-			tmpGallery := "https://cdns-i." + scraperID + ".com/" + sc.SiteID[0:1] + "/" + sc.SiteID[0:4] + "/" + sc.SiteID + "/thumbs/" + size + "_" + x + ".jpg"
-			sc.Gallery = append(sc.Gallery, tmpGallery)
+		if scraperID == "tranzvr" {
+			continue //TranzVR does no longer has preview images
+		}
```
Comment (SpedNSFW) [inline pkg/scrape/wankz.go:82]: Instead of skipping loop iterations, wrap the entire for-loop in an if-statement.

File: pkg/scrape/wankz.go [hunk 3]
```diff
@@ -74,8 +77,11 @@ func WankzVRSite(wg *sync.WaitGroup, updateSite bool, knownScenes []string, out
...
+		tmpGallery := "https://cdns-i." + scraperID + ".com/" + sc.SiteID[0:1] + "/" + sc.SiteID[0:4] + "/" + sc.SiteID + "/thumbs/" + size + "_" + x + ".jpg"
+		sc.Gallery = append(sc.Gallery, tmpGallery)
```
Comment (SpedNSFW) [inline pkg/scrape/wankz.go:84]: You're also removing the indentation here. Don't do that.

File: pkg/scrape/wankz.go [hunk 4]
```diff
@@ -61,10 +61,13 @@ func WankzVRSite(wg *sync.WaitGroup, updateSite bool, knownScenes []string, out
 
 		// Cover URLs
 		for _, x := range []string{"cover", "hero"} {
-			if scraperID == "milfvr" && x == "cover" {
-				continue // MilfVR does not have a "cover" image unlike WankzVR
+			if (scraperID == "milfvr" && x == "cover") || (scraperID == "tranzvr" && x == "hero") {
+				continue // MilfVR does not have a "cover" image & TranzVR doesn't have a "hero" image
```
Comment (SpedNSFW) [inline pkg/scrape/wankz.go:65]:
```suggestion
			if scraperID == "milfvr" && x == "cover" {
				continue // MilfVR does not have a "cover" image unlike WankzVR
			}
			if scraperID == "tranzvr" && x == "hero" {
				continue // TranzVR does not have a "hero" image
```

File: pkg/scrape/wankz.go (hunk 1 above)
Comment (theRealKLH) [reply pkg/scrape/wankz.go]: done

File: pkg/scrape/wankz.go (hunk 2 above)
Comment (theRealKLH) [reply pkg/scrape/wankz.go]: fixed with "break". thanks

File: pkg/scrape/wankz.go (hunk 3 above)
Comment (theRealKLH) [reply pkg/scrape/wankz.go]: done.

**Pull request merged**